src/ArtisanApartmentCommands.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace BuildEmpire\Apartment;
+
+use BuildEmpire\Apartment\Exceptions\SchemaAlreadyExistsException;
+use BuildEmpire\Apartment\Exceptions\SchemaCannotBePublicException;
+use BuildEmpire\Apartment\Exceptions\SchemaDoesntExistException;
+use BuildEmpire\Apartment\Exceptions\SchemaNameNotValidException;
+use BuildEmpire\Apartment\Schema;
+use Illuminate\Support\Facades\Schema as LumenSchema;
+use Carbon\Carbon;
+use BuildEmpire\Apartment\Helpers\ApartmentHelpers;
+
+class ArtisanApartmentCommands
+{
+    const ARTISAN = 1;
+    const WEB = 2;
+    const PUBLIC_SCHEMA = 'public';
+
+    protected $schema = false;
+
+    /**
+     * ArtisanApartmentCommands constructor.
+     * @param \BuildEmpire\Apartment\Schema $schema
+     */
+    public function __construct(Schema $schema)
+    {
+        $this->schema = $schema;
+    }
+
+    /**
+     * Make an apartment's schema and run any existing apartment migrations.
+     *
+     * @param $schemaName
+     * @throws SchemaAlreadyExistsException
+     * @throws SchemaCannotBePublicException
+     * @throws SchemaNameNotValidException
+     */
+    public function makeSchema($schemaName)
+    {
+
+        if (!ApartmentHelpers::isSchemaNameValid($schemaName)) {
```

Comment:
As above, let's put what was discussed in comments, so that people have full oversight. :)